components/appbar/position.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Position
+page_title: AppBar Position
+description: Position settings of the AppBar for Blazor.
+slug: appbar-position
+tags: telerik,blazor,appbar,navbar,position
+published: True
+position: 35
+---
+
+# Position Settings
+
+This article outlines the available AppBar parameters, which control its position.
+
+>note Read the [CSS positioning MDN documentation article](https://developer.mozilla.org/en-US/docs/Web/CSS/position) to get a better understanding of how the AppBar component positioning work.
+
+## Position
+
+The `Position` parameter accepts a member of the `AppBarPosition` enum and sets the `top` and `bottom` CSS properties:
```

Comment:
It's not realistic to expect everyone to go read the CSS article, or to understand it. Better explain in simple terms what each position configuration will achieve. You can still leave the CSS details for the CSS nerds.

Reply:
My idea is to explain the resulting behavior and not just the technical HTML/CSS output of the component. For example - "the component will display at the top of the view port and not move during page scrolling". Something like that.